.travis.yml

```diff
@@ -3,6 +3,10 @@ language: python
 cache:
   directories:
     - $HOME/.cache/pip
+env:
+  global:
+    # Speeds up pip installs and reduces log spam.
+    - PIP_DISABLE_PIP_VERSION_CHECK=1
```

Comment:
> (Boy, Travis should really be setting this by default.)

Yeah I was thinking that too; I've filed https://github.com/travis-ci/travis-ci/issues/5499 now :-)